scene/3d/audio_stream_player_3d.cpp

```diff
@@ -818,6 +850,44 @@ void AudioStreamPlayer3D::_on_bus_renamed(int p_bus_index, const StringName &p_o
 	notify_property_list_changed();
 }
 
+bool AudioStreamPlayer3D::_set(const StringName &p_name, const Variant &p_value) {
+	HashMap<StringName, ParameterData>::Iterator I = playback_parameters.find(p_name);
+	if (I) {
+		ParameterData &pd = I->value;
+		pd.value = p_value;
+		for (Ref<AudioStreamPlayback> &playback : stream_playbacks) {
+			playback->set_parameter(pd.path, pd.value);
+		}
+		if (p_value.is_null()) {
+			playback_parameters.erase(p_name);
+		}
+		return true;
+	}
+
+	return false;
```

Comment:
Same return early comment here.

Reply:
I think we will need to have a discussion about these things because I don't think we need to unify so much in style for this type of code. To me early return is not necessarily more readable always. I will do it for this PR, but I think we need to lax the review requirements a bit.